styles/globals.css

```diff
@@ -25,3 +25,7 @@ a {
 .border-spacing-0 {
   border-spacing: 0;
 }
+
+.color-scheme-dark {
```

Comment:
Don't make a separate class, just use the existing one (Themes.dark, .dark) from ThemeSwitcher